### PR TITLE
Windows And Tables Resprites!

### PR DESCRIPTION
**Pull request by FoxxoTrystan**
# Description

Changes Windows, Tables Sprites, and fixes the state for the airalarm.

Directional windows needed to be updated as we updated the previous ones, tables looked ugly and needed an update, its was provided.

Airalarm has its danger state set to warning, and the other inverted, this fixes it too.

---

# TODO

- [x] AirAlarm State Fix
- [x] Window/Windoor Sprite
- [x] Tables

---

<details><summary><h1>Media</h1></summary>
<p>

image

</p>
</details>

---

# Changelog

:cl:
- tweak: Directional Windows and Tables has gotten new sprites!
- fix: AirAlarm Inverted Alert States has been fixed!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Structures/Furniture/Tables/tables.yml:1]: Me when i touch the "Format Document" button.

**Review by VMSolidus — APPROVED** [review]
Oh those look really nice!

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
Remove single-frame delays from metas and put the delays on one line per direction

**Comment by FoxxoTrystan** [comment]
Really...

Meta File Formatting, you dying that?
Format File.

**Comment by DEATHB4DEFEAT** [comment]
ooh right I can make my edited changes in VSCode into review comments

File: Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json [hunk 1]
```diff
@@ -107,20 +248,44 @@
...
+                    0.3
+                ],
+                [
+                    0.3,
+                    0.3
+                ],
+                [
+                    0.3,
+                    0.3
+                ]
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json:288]:
```suggestion
                [ 0.5, 0.5 ],
                [ 0.5, 0.5 ],
                [ 0.5, 0.5 ],
                [ 0.5, 0.5 ]
            ]
        },
        {
            "name": "fire_3",
            "directions": 4,
            "delays": [
                [ 0.3, 0.3 ],
                [ 0.3, 0.3 ],
                [ 0.3, 0.3 ],
                [ 0.3, 0.3 ]
```

File: Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json [hunk 2]
```diff
@@ -84,12 +216,21 @@
...
         {
             "name": "auth_on",
             "directions": 1,
-            "delays": [[0.1, 0.1]]
+            "delays": [
+                [
+                    0.1,
+                    0.1
+                ]
+            ]
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json:233]:
```suggestion
            "directions": 1
        },
        {
            "name": "auth_on",
            "directions": 1,
            "delays": [ [ 0.1, 0.1 ] ]
```

File: Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json [hunk 3]
```diff
@@ -160,10 +337,38 @@
...
+                    0.2
+                ],
+                [
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2,
+                    0.2
+                ]
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Structures/Wallmounts/air_monitors.rsi/meta.json:371]:
```suggestion
                [ 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 ],
                [ 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 ],
                [ 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 ],
                [ 0.2, 0.2, 0.2, 0.2, 0.2, 0.2 ]
```

File: Resources/Textures/Structures/Furniture/Tables/r_glass.rsi/meta.json [hunk 4]
```diff
@@ -1,163 +1,163 @@
...
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ]
+            ]
```
Comment (DEATHB4DEFEAT) [inline Resources/Textures/Structures/Furniture/Tables/r_glass.rsi/meta.json:160]:
```suggestion
            "name": "full"
        },
        {
            "name": "state_0",
            "directions": 4
        },
        {
            "name": "state_1",
            "directions": 4
        },
        {
            "name": "state_2",
            "directions": 4
        },
        {
            "name": "state_3",
            "directions": 4
        },
        {
            "name": "state_4",
            "directions": 4
        },
        {
            "name": "state_5",
            "directions": 4
        },
        {
            "name": "state_6",
            "directions": 4
        },
        {
            "name": "state_7",
            "directions": 4
```

**Comment by DEATHB4DEFEAT** [comment]
image
guh

**Comment by FoxxoTrystan** [comment]
image

Just for file format this is pain, gosh.
What Format File VSCode Setting are you using so you dont trouble me with this ever again?

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**